packages/cli/src/commands/dev.ts

```diff
@@ -98,7 +98,7 @@ export default class Dev extends Command {
       cwd: join(__dirname, '..', '..', 'assets'),
       composeOptions: ['-p', 'liteflow'],
     })
-    await this.waitToBeReady()
+    await this.waitToBeReady(new Date().setMinutes(new Date().getMinutes() + 1))
```

Comment:
`waitToBeReady` expects a number, not a Date.
You can do something like:
```suggestion
    await this.waitToBeReady(Date.now() + 60 * 1000) // timeout in 60sec
```

Reply:
updated.